openslides/motions/views.py

```diff
@@ -66,6 +66,16 @@ def check_view_permissions(self):
             result = False
         return result
 
+    def retrieve(self, request, *args, **kwargs):
+        """
+        Customized view endpoint to retrieve a motion.
+
+        Adds the allowed actions for the motion.
+        """
+        response = super().retrieve(request, *args, **kwargs)
+        response.data['allowed_actions'] = self.get_object().get_allowed_actions(request.user)
+        return response
```

Comment:
@emanuelschuetze asked me about a quick and dirty hack to get the allowed actions from the server. I said that we might want to hold the logic on client side but we do not have too much time at the moment. Read-only field in the serializer is not easy because we do not have the request user there. You can try ...